### PR TITLE
#185218941: A buyer should see recommended products

### PR DESCRIPTION
**Pull request by bergad37**
## What does this PR do?

This section should display several recommended products

## Description of Task to be completed?

- Given that the buyer is viewing a product page:
- Then they should see a section labeled "You may also like" or "Similar"
- This section should display several recommended products

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in the root directory of the application.

- Start the application development server: `npm start

## What are the relevant pivotal tracker/Trello stories?
[#185218941](https://www.pivotaltracker.com/story/show/185218941)

### CODE REVIEW

File: src/containers/HomePopularProducts.jsx [hunk 1]
```diff
@@ -53,13 +51,16 @@ const PopularProducts = () => {
             name={wishlist.product.name}
             price={wishlist.product.price}
             pId={wishlist.productId}
-            image={wishlist.product.image[0]}
+            image={wishlist.product.image[random(wishlist.product.image.length)]}
```
Comment (nishimweprince) [inline src/containers/HomePopularProducts.jsx:54]: This is brilliant 🔥

File: src/containers/HomeBrowseCategory.jsx [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-    rating: {
-      rating: 3,
-      count: 5
-    },
-    className:
-      'home_category_product_card flex flex-col justify-between my-4 w-fit max-w-1/5'
-  };
+  const[getAllProducts]=useLazyGetAllProductsQuery();
+  const [populars,setPopulars]=useState([]);
+  const category='BEAUTY';
```
Comment (nishimweprince) [inline src/containers/HomeBrowseCategory.jsx:12]: I suggest that you use the categories obtained from our API, and use the getCatName function defined in [`src/utils`](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/ft-view-recommended-products-185218941/src/utils/categories.js#L1C1-L6C3) to get the category name dynamically using its (category) id.

The function looks like this:
```jsx
const getCatName = (id, arr) => {
  const category = arr.find((item) => {
    return item.id === id;
  });
  return category ? category.name : 'Category';
};
```

**Review by Dyves1 — APPROVED** [review]

**Review by nishimweprince — COMMENTED** [review]
I added a comment for your consideration!

**Review by nishimweprince — APPROVED** [review]
Excellent work!!

**Review by irakevine — APPROVED** [review]
Great work!

**Review by niyontwali — APPROVED** [review]

**Pull request merged**